README.md

```diff
@@ -252,6 +263,86 @@ addEventListener('DOMContentLoaded', function() {
 
 _**Note:** it's safe to use module scripts in legacy browsers because unknown script types are ignored._
 
+## Bundle options
+
+The `web-vitals` library includes a base set of libraries as well as a polyfill script to improve [browser support](#browser-support) for some of the metrics (where possible).
+
+The library is released as several different build versions, allowing developers to chose the version that best meets their needs or integrates with their architecture.
+
+<table>
+  <tr>
+    <td width="35%">
+      <strong>Filename</strong> <em>(all within <code>dist/*</code>)</em>
+    </td>
+    <td><strong>Export</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td><code>web-vitals.full.js</code></td>
+    <td><code>pkg.module</code></td>
+    <td>
+      <p>An ES module bundle of all metric functions, including a minimal polyfill to support back/forward cache restores.</p>
+      Using the "full" bundle is the simplest way to consume this library out of the box.
+    </td>
+  </tr>
+  <tr>
+    <td><code>web-vitals.full.umd.js</code></td>
+    <td><code>pgk.main</code></td>
+    <td>
+      A UMD version of <code>web-vitals.full.js</code> (exposed on the <code>window.webVitals.*</code> namespace).
+    </td>
+  </tr>
+  <tr>
+    <td><code>web-vitals.base.js</code></td>
+    <td>--</td>
+    <td>
+      <p>An ES module bundle of the <code>web-vitals</code> library without any polyfills included.</p>
+      Use this bundle if (and only if) you've also added the <code>polyfill.js</code> script to the <code>&lt;head&gt;</code> of your pages. See <a href="#how-to-use-the-polyfill">how to use the polyfill</a> for more details.
+    </td>
+  </tr>
+  <tr>
+    <td><code>polyfill.js</code></td>
+    <td>--</td>
+    <td>
+      A set of small polyfills that expands browser supports (including back/forward cache restores) and fills in some measurements gaps. See <a href="#how-to-use-the-polyfill">how to use the polyfill</a> for more details.
+    </td>
+  </tr>
+</table>
+
+### Which bundle is right for you?
+
+Most developers will generally want to use the "full" bundle (either the ES module or UMD version, depending on your build system), as it's the easiest to use out of the box and integrate into existing build tools.
+
+However, there are a few good reasons to consider using the "base" version along with the `polyfill.js` script. For example:
+
+- FID can be measured in all browsers.
+- FCP, FID, and LCP will be more accurate in some cases (since the polyfill detects the page's initial `visibilityState` earlier).
+
+Also, the minimal polyfill to support back/forward cache restores that is found in the "full" version is largely the same as the code used in the `polyfill.js` script—just split out. This means that using the polyfill gets you wider browser support and more accurate results with almost no increased code cost (just increased implementation complexity).
```

Comment:
This confused me a bit :)

There's a built-in polyfill in the "full" version that is pretty much the same as _polyfill.js_, but using "base" + _polyfill.js_ provides better browser support and accuracy? Maybe add a one-liner to explain why?

Reply:
Now that I understand, I think the term "full" is what makes it confusing. It makes it seem like it has everything (back-forward cache restores **and** FID polyfill)

Changing it to something else should make things clearer 🚀